server/uploadHandler/src/service/dynamodbService.ts

```diff
@@ -0,0 +1,25 @@
+import { PutItemInput } from "aws-sdk/clients/dynamodb";
+import { v4 as uuidv4 } from "uuid";
+
+import { putItem } from "./dynamodbClient";
+
+const tableName = process.env.JOB_TABLE as string;
+
+export const addNewJobToDb = async (
+  email: string,
+  videoFiles: string[],
+  imageFiles: string[]
+): Promise<void> => {
+  const newJobItem = {
+    TableName: tableName,
+    Item: {
+      id: uuidv4,
+      email,
+      videoFiles,
+      imageFiles,
+      updatedAt: new Date().toISOString(),
```

Comment:
@beining00 Is the aim of this PR to finalise the database schema? Or will that be done later?
One thing missing from this is the status of each image/video

Reply:
I wrote this just to give an example of how to update the database. The actual fields in the table can be changed later on.